.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+[STORY_LINK]
+
+## Why?
+
+Describe why you do this PR.
```

Comment:
```suggestion
To describe the reason for creating this **PR**.
```

Reply:
Please discuss this in here: https://github.com/Cobda/cobda-web/pull/2#discussion_r559502433 and remove **this** duplicate.